src/features/ExploreView/ExploreMenu.js

```diff
@@ -143,17 +149,32 @@ const ExploreMenu = ({ navigation }) => {
                 })}
 
               </ScrollView>
-              {/* onPress 'create a session', it will direct to the QR code -  */}
-              <View styles={styles.button}>
+              {/* MENU scrollable View */}
+              <View style={styles.menuContainer}>
+
+
+
+                {/* onPress 'create a session', it will direct to the QR code -  */}
+
                 <Button
                   icon='account-plus'
                   mode="contained"
                   color={colors.brand.beachLight}
-                  onPress={() => createSessionHandler()}>
+                  style={{ width: 250, borderRadius: 15, height: 38 }}
+                  onPress={() => {
+                    createSessionHandler();
+                    navigation.navigate('CurrentSession', { previous: 'create a session' });
```

Comment:
Kindly delete this line 166. It's already handled in createSessionHandler() method.